src/app/shared/item/item-versions/item-versions-delete-modal/item-versions-delete-modal.component.ts

```diff
@@ -1,12 +1,18 @@
-import { Component } from '@angular/core';
+import { Component, Output } from '@angular/core';
 import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { Subject } from 'rxjs';
 
 @Component({
   selector: 'ds-item-versions-delete-modal',
   templateUrl: './item-versions-delete-modal.component.html',
   styleUrls: ['./item-versions-delete-modal.component.scss']
 })
 export class ItemVersionsDeleteModalComponent {
+  /**
+   * An event fired when the cancel or confirm button is clicked, with respectively false or true
+   */
+  @Output()
+  response: Subject<boolean> = new Subject();
```

Comment:
This modal was failing in tests & I'd lifted this from some previous modal that _wasn't_ breaking, probably didn't think about it since I didn't really use it as an output...
Found three such cases & updated them to `EventEmitter`s.